test/e2e/test_resource.py

```diff
@@ -2,22 +2,25 @@
 import unittest
 
 from knora.dsplib.models.connection import Connection
+from knora.dsplib.models.helpers import BaseError
 from knora.dsplib.models.permission import PermissionValue, Permissions
 from knora.dsplib.models.resource import ResourceInstanceFactory
 from knora.dsplib.models.sipi import Sipi
 from knora.dsplib.models.value import KnoraStandoffXml, make_value
 
 
 class TestResource(unittest.TestCase):
+    con = Connection('http://0.0.0.0:3333')
 
-    def setUp(self) -> None:
-        """
-        is executed before all tests; sets up a connection and logs in as user root
-        """
-        self.con = Connection('http://0.0.0.0:3333')
-        self.con.login('root@example.com', 'test')
```

Comment:
I need the connection for several methods, so the old `setUp()` method is not so good.